### PR TITLE
Pull Requests for Data

### LINKED ISSUE #7998: Pull Request for Data
Have you considered implement Pull Request feature on LakeFS. So people can review data changes before merging into main branch.

### LINKED ISSUE #8182: PRfD - WebUI - Add Playwright tests
Add a test in Playwright to verify that PRfD works as expected.

### PR DESCRIPTION
**Pull request by itaigilo**
Closes #7998.
Closes #8182.

-----

## Change Description

Add tests,
And enable the feature in the WebUI.

### CODE REVIEW

File: webui/src/lib/components/repository/tabs.jsx
Comment (itaigilo) [inline webui/src/lib/components/repository/tabs.jsx:1]: This reveals PRfD to everyone.

File: webui/src/pages/repositories/repository/pulls/pullsList.jsx
Comment (itaigilo) [inline webui/src/pages/repositories/repository/pulls/pullsList.jsx:1]: Added classes to make elements more discoverable.

File: webui/test/e2e/common/quickstart.spec.ts
Comment (itaigilo) [inline webui/test/e2e/common/quickstart.spec.ts:1]: For non `"pull requests"` tests -
It's only about auto-formatting, plus removing some redundant `await`s (as wisely suggested by IntelliJ).

File: webui/src/lib/components/repository/tabs.jsx [hunk 1]
```diff
@@ -60,14 +57,10 @@ export const RepositoryNavTabs = ({ active }) => {
             <Link active={active === 'tags'} href={`/repositories/${repoId}/tags`} component={NavItem}>
                 <TagIcon/> Tags
             </Link>
-            {
-                // TODO (gilo): this is temp, until PRfD will be ready
-                showPulls &&
```
Comment (guy-har) [inline webui/src/lib/components/repository/tabs.jsx:65]: 😄

**Review by guy-har — APPROVED** [review]

**Pull request merged**